cmd/deployment_create.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 Northern.tech AS
+//
+//	Licensed under the Apache License, Version 2.0 (the "License");
+//	you may not use this file except in compliance with the License.
+//	You may obtain a copy of the License at
+//
+//	    http://www.apache.org/licenses/LICENSE-2.0
+//
+//	Unless required by applicable law or agreed to in writing, software
+//	distributed under the License is distributed on an "AS IS" BASIS,
+//	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//	See the License for the specific language governing permissions and
+//	limitations under the License.
+package cmd
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/mendersoftware/mender-cli/client/deployments"
+	"github.com/mendersoftware/mender-cli/log"
+)
+
+const (
+	argDeploymentName = "deployment-name"
+)
+
+var deploymentCreateCmd = &cobra.Command{
+	Use:   "create [flags] DEPLOYMENT_NAME ARTIFACT_NAME DEVICE_LIST",
```

Comment:
This should be documented better, especially device list. Like where it can be found, and whether the list is comma separated or not. I know this comes from the API, but people don't typically look there.